### ISSUE TITLE
Ensure that .conf files are not stepped on during 'apt upgrade'

### ISSUE THREAD

**Issue by Allan-N** [AllStarLink/ASL3#114]
We would like to make changes to the default/template configuration (.conf) files.  But, we also want to ensure that any of our software updates do not, inadvertently, step on an existing configuration.  Working configurations need to keep working.

For now, "I" think we need to just leave any existing configuration file as-is.  While not required at this time, having a "hook" that would allow us to exec a script/program that would help migrate/modify an existing configuration file would be a plus.

**Comment by jxmx** [comment]
Move to ASL3 package

**Comment by InterLinked1** [comment]
Don't Debian packages already have provisions for managing this? When upgrading, I often get a message "the package has a newer config, what would you like to do?" with options to either keep my existing file or use the new one.

**Comment by jxmx** [comment]
Yes it does. There's a number of people who keep saying "Y" to random files to replace them breaking their configuration. There's an associated pull I just make that will unconditionally answer "N" (keep) to all package installs.

**Comment by jxmx** [comment]
Fixed by #115
